styleguide.config.js

```diff
@@ -28,6 +28,10 @@ module.exports = {
           name: 'Basic example',
           content: 'examples/Basic/Readme.md'
         },
+        {
+          name: 'Dragging folders example',
```

Comment:
Can this be pushed below, please? It should be the last example since it's the most advanced one.